### DISCUSSION TITLE
Choice of the icon for the extension

### DISCUSSION THREAD

**Discussion by FALLAI-Denis** [zokugun/vscode-explicit-folding#61] [General]
Hi @daiyam 

I noticed that you recently use the characters sequence `{}` to define the icon associated with the extension (and other child extensions ;-) ).

However, these characters can create confusion with extensions of type Mustache or Handlebars which historically manipulate JSON text, hence the reference to the characters `{}`.

I don't have a suggestion for the moment for the definition of an icon that reflects the nature of the extension, perharps a `+/-` icon? but can we open a debate here on this subject?

Can be inspired by the icons found by googling "folding icon" or "expand reduce icon"...

For information, for old computer technologies related to the mainframe, of which the COBOL language is part, we generally associate the image of a T-Rex (heavy, large, not very agile, but so powerful) rather than of a triceratops ;-).

**Comment by daiyam** [comment]
The `{}` come from my main use of emacs folding: `{{{ ... }}}` but I can see it a bit confusing for other users.
Maybe an origami, a paper plane...

For the Cobol logo, I've done a google search and only found a triceratops. Yep, `trex cobol` gives some cool images...

**Comment by FALLAI-Denis** [comment]
For COBOL icon, can google "grace hopper icon" for image. 
Grace Hopper is mother of COBOL language (in 1959).

**Comment by daiyam** [comment]
What about:
![icon2 copy 9](https://user-images.githubusercontent.com/587742/126060311-26105eaa-fd56-40fd-b7e4-cd48765dbb97.png)

**Reply by FALLAI-Denis** [reply]
Great !
